packages/core/src/utils.ts

```diff
@@ -1404,3 +1404,21 @@ export function flatMapFilter<T, U>(arr: T[] | undefined, fn: (value: T, idx: nu
   }
   return result;
 }
+
+/**
+ * Returns the escaped HTML string of the input string.
+ * @param unsafe - The unsafe HTML string to escape.
+ * @returns The escaped HTML string.
+ */
+export function escapeHtml(unsafe: string): string {
```

Comment:
This function already existed in the `generator` package.  Here I'm moving it from `generator` to `core` so we can access it from `server`.

If we're concerned about bloating `core`, then we could also just duplicate this in `server`.

Reply:
It's fine to put it in core I think 👍